dcos_launch/platforms/aws.py

```diff
@@ -139,6 +138,12 @@ def get_all_stacks(self):
         for stack in self.get_service_resources('cloudformation', 'stacks'):
             yield CfStack(stack.stack_name, self)
 
+    @retry_boto_rate_limits
+    def get_all_buckets(self):
+        """Get all S3 buckets in all regions."""
+        for bucket in self.get_service_resources('s3', 'buckets'):
```

Comment:
@cprovencher see the example below:
`yield from self.get_service_resources('s3', 'buckets')`
https://github.com/dcos/dcos-launch/pull/98/files#diff-7a39b8bfb78505e409951f5d198d05a2R150